### ISSUE TITLE
The installed mimiciv data will be deleted automatically

### ISSUE THREAD

**Issue by hobewoos** [MIT-LCP/mimic-iv#55]
I had installed the mimiciv database, and it displayed normally on postgresapp.  But when I log out of psql and the external hard disk, the next time I log in again, the data of mimiciv database will be automatically deleted, leaving only the created tables with 0 rows.
I have reinstalled 4 times and encountered the same problem.......

**Comment by Qinyu-Zhao0522** [comment]
Hi!

I am not a student or professor in MIT-LCP but I encountered a similar problem recently. When I was working on calculating pivoted SOFA, the sql process crashed because it's out of memory. After my computer restored, I found that all data in my tables were lost, leaving 0 rows like yours. I checked the create.sql and found that all tables in MIMIC-IV are unlogged. All data in an unlogged table may be lost if the database process crashed or is killed. I suppose that's why your data were lost after you log out.

Try to modify the create.sql like 'CREATE TABLE ... AS', and reinstall your database. I hope this help.

Qinyu
